hera_xblocks/end_survey/end_survey/end_survey.py

```diff
@@ -77,7 +81,9 @@ def result_summary_count(self):
                 }
             }
         }
-        # Count procentage:
+        if not self.result_summary:
```

Comment:
you won't get into loop bellow if `self.result_summary` is an empty. I think it's redundant.

Reply:
There will be a blank page for staff if not to do this check and nobody answered yet.

Reply:
fixed